multiversx_sdk_cli/cli_deps.py

```diff
@@ -34,16 +36,63 @@ def install(args: Any):
 
 def check(args: Any):
     name: str = args.name
-    module = dependencies.get_module_by_key(name)
-    tag_to_check: str = config.get_dependency_tag(module.key)
+
+    if name == "all":
+        all_dependencies = dependencies.get_all_deps()
+        missing_dependencies: List[Tuple[str, str]] = []
+
+        for dependency in all_dependencies:
+            tag_to_check: str = config.get_dependency_tag(dependency.key)
+            is_installed = check_module_is_installed(dependency, tag_to_check)
```

Comment:
This does not handle `rust` the complete way, as done below.

Reply:
Fixed